thingiverse/ThingiverseService.py

```diff
@@ -316,14 +377,14 @@ def _onDownloadFinished(self, file_bytes: bytes, file_name: str) -> None:
         self._is_downloading = False
         self.downloadingStateChanged.emit()
 
-    def _onQueryFinished(self, things: List[JSONObject]) -> None:
+    def _onQueryFinished(self, objects: List[JSONObject]) -> None:
```

Comment:
Not sure if objects is a better name, it's quite generic (in relation to Python).